ff-asn-apply-filters/index.html

```diff
@@ -194,10 +233,26 @@
 
         <div slot="groupCaption"><div>{{group.name}}</div></div>
 
-        <div data-container="removeFilter">[x] Reset Filter</div>
-      </ff-asn-group>
+        <div data-container="showMore">
+          <svg viewBox="0 0 330 330"><use xlink:href="#icon-angle-down"></use></svg>
+          <span>Show More</span>
+        </div>
+        <div data-container="showLess">
+          <svg viewBox="0 0 330 330"><use xlink:href="#icon-angle-up"></use></svg>
+          <span>Show Less</span>
+        </div>
 
+        <div data-container="removeFilter">
+          <svg viewBox="3 3 10 10"><use xlink:href="#icon-cross"></use></svg>
+          <span>Reset Filter</span>
+        </div>
+      </ff-asn-group>
       <ff-asn-group filter-style="DEFAULT">
+        <div slot="filterSearch">
+          <input placeholder="Search group...">
+          <svg class="svg-search" viewBox="0 0 30 30"><use xlink:href="#search"></use></svg>
+        </div>
+
         <ff-asn-group-element>
           <div slot="selected">
             <span class="circle"></span> {{element.name}}&nbsp;{{group.unit}}
```

Comment:
Fixed. I think it might be even considered as a good practice to bind `element.name` with triple curly braces since we have the searchable groups feature.

Reply:
Also, the "Apply Filters" demo doesn't seem to be working properly with the searchable asn groups. Not sure whether we should address this issue or just keep in mind that it has such limitation.

Reply:
I think we can address it separately but I won't object either if you feel like fixing it now.